interface/wx/event.h

```diff
@@ -3644,6 +3644,251 @@ class wxScrollEvent : public wxCommandEvent
     void SetPosition(int pos);    
 };
 
+
+
+/** @class wxGestureEvent
+    This is the base class for all supported gesture events.
+
+    @note Gesture events are not generated by default, you must call
+          wxWindow::EnableTouchEvents() with the appropriate parameter to
+          request their generation.
+
+    @library{wxcore}
+    @category{events}
+
+    @see wxPanGestureEvent, wxZoomGestureEvent, wxRotateGestureEvent
+
+    @since 3.1.1
+*/
+class wxGestureEvent : public wxEvent
```

Comment:
How do you intend to integrate raw touch events into the wxEvent tree?

Each touch event has: 
* a point
* a ID to track the movement of indivual fingers
* if it is start, end or just a move [and cancel on some plattform].
* primary indicator [= considered as mouse pointer].

Reply:
So it looks like the current `wxGestureEvent` could be used as the base class for the new `wxMultiTouchEvent`, right? We'd just need to rename it to, say, `wxTouchEventBase` and inherit `wxMultiTouchEvent` and (now trivial) `wxGestureEvent` from it, what do you think?